runtime/auth/aws-signing-standard/common/src/aws/smithy/kotlin/runtime/auth/awssigning/standard/StandardAwsSigner.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package aws.smithy.kotlin.runtime.auth.awssigning.standard
+
+import aws.smithy.kotlin.runtime.auth.awscredentials.Credentials
+import aws.smithy.kotlin.runtime.auth.awssigning.AwsSigner
+import aws.smithy.kotlin.runtime.auth.awssigning.AwsSigningAlgorithm
+import aws.smithy.kotlin.runtime.auth.awssigning.AwsSigningConfig
+import aws.smithy.kotlin.runtime.auth.awssigning.AwsSigningResult
+import aws.smithy.kotlin.runtime.http.request.HttpRequest
+import aws.smithy.kotlin.runtime.time.TimestampFormat
+
+/** The standard implementation of [AwsSigner] */
+val StandardAwsSigner: AwsSigner = StandardAwsSignerImpl()
```

Comment:
Yeah, we can do that. I wanted to call it "Default" originally but then couldn't easily use `default` as a namespace because it's a Java keyword. But if we just put it in the parent (i.e., `awssigning`) namespace that takes care of that.